ios_ui/MovieRec/AppDelegate.swift

```diff
@@ -16,6 +17,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        
+        if UserDefaults.standard.string(forKey: "userID") != nil {
```

Comment:
I guess I made this change in the other PR. Is optional binding convention even if you don't use the value?